test/travis_test/build_2_container.sh

```diff
@@ -0,0 +1,105 @@
+    mkdir -p /etc/openvswitch && \
+    ovsdb-tool create /etc/openvswitch/conf.db /usr/local/share/openvswitch/vswitch.ovsschema && \
+    mkdir -p /var/run/openvswitch && \
+    ovsdb-server --remote=punix:/var/run/openvswitch/db.sock --remote=db:Open_vSwitch,Open_vSwitch,manager_options --pidfile --detach --log-file && \
+    ovs-vsctl --no-wait init && \
+    ovs-vswitchd --pidfile --detach --log-file && \
+    ovs-vsctl add-br br-int -- set bridge br-int datapath_type=netdev && \
+    ovs-vsctl add-br br-tun -- set bridge br-tun datapath_type=netdev && \
+    ovs-vsctl \
+    -- add-port br-int patch-tun \
+    -- set interface patch-tun type=patch options:peer=patch-int \
+    -- add-port br-tun patch-int \
+    -- set interface patch-int type=patch options:peer=patch-tun && \
+    ovs-ofctl add-flow br-tun "table=0, priority=1,in_port="patch-int" actions=resubmit(,2)" && \
+    ovs-ofctl add-flow br-tun "table=2, priority=0 actions=resubmit(,22)"'
+
+    docker exec aca_CHILD bash -c ' \
+    ovs-ctl --delete-bridges stop && \
+    rm -rf /etc/openvswitch /usr/local/etc/openvswitch && \
+    mkdir -p /var/log/openvswitch && \
+    mkdir -p /etc/openvswitch && \
+    ovsdb-tool create /etc/openvswitch/conf.db /usr/local/share/openvswitch/vswitch.ovsschema && \
+    mkdir -p /var/run/openvswitch && \
+    ovsdb-server --remote=punix:/var/run/openvswitch/db.sock --remote=db:Open_vSwitch,Open_vSwitch,manager_options --pidfile --detach --log-file && \
+    ovs-vsctl --no-wait init && \
+    ovs-vswitchd --pidfile --detach --log-file && \
+    ovs-vsctl add-br br-int -- set bridge br-int datapath_type=netdev && \
+    ovs-vsctl add-br br-tun -- set bridge br-tun datapath_type=netdev && \
+    ovs-vsctl \
+    -- add-port br-int patch-tun \
+    -- set interface patch-tun type=patch options:peer=patch-int \
+    -- add-port br-tun patch-int \
+    -- set interface patch-int type=patch options:peer=patch-tun && \
+    ovs-ofctl add-flow br-tun "table=0, priority=1,in_port="patch-int" actions=resubmit(,2)" && \
+    ovs-ofctl add-flow br-tun "table=2, priority=0 actions=resubmit(,22)"'
+
+  echo "    --- running alcor-control-agent ---"
+  # sends output to null device, but stderr to console 
+  docker exec aca_PARENT bash -c "nohup /mnt/host/code/build/bin/AlcorControlAgent -d > /dev/null 2>&1 &"
+  docker exec aca_CHILD bash -c "nohup /mnt/host/code/build/bin/AlcorControlAgent -d > /dev/null 2>&1 &"
+
+  echo "    --- Running unit test cases ---"
+  docker exec aca_PARENT bash -c "cd /mnt/host/code/build/tests && ./aca_tests && ./gs_tests"
+  docker exec aca_CHILD bash -c "cd /mnt/host/code/build/tests && ./aca_tests && ./gs_tests"
+
+  mv -f $code_dir/src/ovs/aca_ovs_l2_programmer.cpp.bak $code_dir/src/ovs/aca_ovs_l2_programmer.cpp
+  # Build alcor control agent
+  echo "--- building alcor-control-agent post test ---"
+  docker exec aca_PARENT bash -c "cd /mnt/host/code && make"
+  docker exec aca_CHILD bash -c "cd /mnt/host/code && make"
+
+fi
```

Comment:
this is a great start on travis setup, please look into why travis is returning the below error:

Successfully tagged aca_build0:latest
Error: No such container: aca_PARENT
b18118e443da96d524cec244ae64f8d6eb1edbe07c66b83c0ba50a9ec45e38ef
aca_PARENT
Error: No such container: aca_CHILD
28f0d8eb26263559049b97d326d3461a2b8512e1bd970cad315c014f72fc9991
aca_CHILD
./test/travis_test/build_2_container.sh: line 106: syntax error: unexpected end of file
The command "./test/travis_test/build_2_container.sh 2_port_test" exited with 2.
Done. Your build exited with 1.

Reply:
OK, thanks for your advice. and I will solve the bug as soon as possible, and i wonder if the logic of the travis script is ok ?
(first build two container, then run DISABLED_2_ports_CREATE_test_traffic_CHILD and DISABLED_2_ports_CREATE_test_traffic_PARENT test case)?

Reply:
The logic looks good. Let finish up the remaining items and then we can merge this in.